### PR TITLE
Timeseries fix

### PR DESCRIPTION
**Pull request by kat-dolb**
Fixed missing timeseries plot for global VM totals, and added job status plots for individual aliases (when enabled on status page)

### CODE REVIEW

File: data_collectors/general/timeseriesPoller.py [hunk 1]
```diff
@@ -181,9 +198,7 @@ def timeseries_data_transfer():
                         'ram_native_foreign',
                         'slot_count',
                         'slot_core_count',
-                        'slot_idle_core_count',
-                        'volume_gigs_max',
```
Comment (colsond) [inline data_collectors/general/timeseriesPoller.py]: Is there a reason these 2 volume metrics were removed?

File: web_frontend/cloudscheduler/csv2/templates/csv2/status.html [hunk 2]
```diff
@@ -408,16 +453,15 @@
 
                             <th class="col-spacer">&nbsp;</th>
 
-                            <th class="total-style" colspan="4">
```
Comment (colsond) [inline web_frontend/cloudscheduler/csv2/templates/csv2/status.html]: Seems like more volume stuff was removed here, I don't remember if we had a reason for doing so.

File: web_frontend/cloudscheduler/csv2/templates/csv2/status.html [hunk 3]
```diff
@@ -448,17 +492,7 @@
                                 {% if cloud.display >= 9 %}
                                     <td class="bold-style" onclick="toggle_id('expand-{{cloud.group_name}}-{{cloud.cloud_name}}')"><div class="float-left">Totals</div>
                                 {% elif cloud.enabled > 0 %}
-                                    {% if cloud.app_credentials_expiry and cloud.current_time %}
```
Comment (colsond) [inline web_frontend/cloudscheduler/csv2/templates/csv2/status.html]: Seems like you may have been working with an older version of this file as your merge request removes a lot of the stuff added in 2.9.0, the application credential stuff and the volume stuff should be in the final version. You may want to merge 2.9.0 into your branch first then make a new pull request with your changes.

**Review by colsond — CHANGES_REQUESTED** [review]
The new code introduced looks great but it seems you may have not pulled in the most recent chagnes from 2.9.0 before making the pull request. I suggest pulling 2.9 down into your working branch, resolving the merge and then updating the pull request.

File: web_frontend/cloudscheduler/csv2/templates/csv2/status.html (hunk 3 above)
Comment (kat-dolb) [reply web_frontend/cloudscheduler/csv2/templates/csv2/status.html]: stable-2.9 has been merged and the missing volume stuff added manually

**Review by colsond — APPROVED** [review]

**Pull request merged**